docs/instructors.md

```diff
@@ -231,3 +231,9 @@ For the gradebook calculations, submissions are classified as one of three types
 ## Releasing Grades
 
 Manually assigned grades are by default not released, and therefore not visible to students. You can release grades on an individual basis while grading, or release all available grades in bulk by using the "Release all grades" option. You can also reverse this process using the "Withdraw all grades" option. (The word "withdraw" is perhaps unfortunate. No grades are ever deleted. They are simply withdrawn from the student's view.)
+
+## Resetting a student's password
+
+An instructor is able to trigger the reset of the student's password by selecting `Manage Users` from the `Manage Autolab` dropdown if a student does not have access to a registered email address and forgets their password. The instructor can click on a student from the generated `Users List` and click on `Change User's Password` to reset the student's password.
```

Comment:
Only admins can trigger password reset, so this isn't accurate. Right now there isn't a good place to put admin features, but for now could you put this under the "Features" section under a new category?
![Screenshot 2023-12-01 at 12 03 10 PM](https://github.com/autolab/Autolab/assets/25730111/086d8421-160e-42ad-a5fa-4cd613c9c409)

Reply:
Okay, I moved it to a new category called admin features.

Reply:
You need to add a link to the new md file in `mkdocs.yml` (look at lines 21-35) for it to show up properly. Have you been running mkdocs locally to check your changes to docs?

Reply:
Okay, added the link.